reproject/mosaicking/wcs_helpers.py

```diff
@@ -6,12 +6,12 @@
 from astropy.wcs.utils import (celestial_frame_to_wcs, pixel_to_skycoord, proj_plane_pixel_scales,
                                skycoord_to_pixel, wcs_to_celestial_frame)
 
-from ..utils import parse_input_data
+from ..utils import parse_input_shape
 
 __all__ = ['find_optimal_celestial_wcs']
 
 
-def find_optimal_celestial_wcs(input_data, frame=None, auto_rotate=False,
+def find_optimal_celestial_wcs(input_shapes, frame=None, auto_rotate=False,
```

Comment:
I'm hesitant to change the argument name as I've seen cases where the first arg is passed as a kwarg - can you change the name back?